Core/SiteRating.swift

```diff
@@ -55,13 +55,23 @@ public class SiteRating {
             
         return nil
     }
-    
-    public var containsMajorTracker: Bool {
-        return trackersDetected.contains(where: { $0.key.fromMajorNetwork } )
+
+    public var majorTrackersDetected: Int {
+        return majorTrackers(trackers: trackersDetected)
```

Comment:
This is really a count of the unique major tracker networks detected (not a total). Maybe rename to `uniqueMajorTrackerNetworksDetected?